src/js/queries/informationPageFragment.js

```diff
@@ -21,6 +21,47 @@ const informationPageFragment = `
         title
       }
     }
+    events {
```

Comment:
Can you create an event fragment so we don't have to copy/paste it?

Reply:
I will try, i may have to rework allPages because they dont like the same fragment being imported multiple times

Reply:
Ah I forgot about that. Okay, we can backlog it if its a whole thing